optimum/intel/openvino/trainer.py

```diff
@@ -78,6 +92,7 @@
 
 set_log_level(logging.ERROR)
 logger = logging.get_logger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
With loglevel INFO  "Statistics of the sparsified model" printed almost 3000 lines when I tested this. That drowns out all other information in the terminal.

Could we handle that differently? For example, send that to a logfile?

Reply:
@helena-intel  see https://github.com/huggingface/optimum-intel/pull/150/commits/9e2f4ee7385dcb98256fcac6fe1445729a39874c .